src/views/UserSearch/Filters.js

```diff
@@ -94,27 +105,45 @@ export default class Filters extends React.Component {
           displayClearButton
           id="users-filter-accordion-patron-group"
           header={FilterAccordionHeader}
-          label={<FormattedMessage id="ui-users.information.patronGroup" />}
+          label={formatMessage({ id: 'ui-users.information.patronGroup' })}
           separator={false}
           onClearFilter={() => clearGroup('pg')}
         >
           <CheckboxFilter
-            dataOptions={this.getValuesFromResources('patronGroups', 'group')}
+            dataOptions={this.getValuesFromResources('patronGroups', 'group', 'group')}
```

Comment:
@BogdanDenis would you mind changing this to `this.getValuesFromResources('patronGroups', 'group', 'id')`

Reply:
This will restore the behavior of #1258 that was inadvertently removed in #1441.